js/start.js

```diff
@@ -387,11 +397,30 @@ function start() {
 
     app.walletBalance = new WalletBalance();
 
+    app.searchProviders = new SearchProvidersCol();
+
     onboardIfNeeded().done(() => {
       fetchStartupData().done(() => {
         app.pageNav.navigable = true;
         app.pageNav.setAppProfile();
         app.loadingModal.close();
+
+        // set the default search providers.
+        app.searchProviders.add(getDefaultSearchProviders());
+        // if active and default providers aren't set, set them now.
+        if (!app.searchProviders.activeProvider) {
+          app.searchProviders.setProvider(app.searchProviders.at(0), 'active');
```

Comment:
Instead of calling `setProvider`, would we want to use the setters here?

`app.searchProviders.activeProvider = app.searchProviders.at(0);`